src/DynamoApplications/StartupUtils.cs

```diff
@@ -273,6 +275,13 @@ public static DynamoModel MakeModel(bool CLImode, string asmPath = "", string ho
         /// <returns></returns>
         public static DynamoModel MakeModel(bool CLImode, string asmPath = "", HostAnalyticsInfo info = new HostAnalyticsInfo())
         {
+            IPathResolver pathResolver = CreateIPathResolver(false, string.Empty, string.Empty, string.Empty);            
```

Comment:
Have you tested in DynamoRevit yet? Still do not think this would work...

Reply:
These string.empty params later will cause certain folder property to be missing. For sandbox it is OK but for integration case, we get these from integration code